doc/builtin_entities.txt

```diff
@@ -0,0 +1,96 @@
+# Builtin Entities
+Minetest registers two entities by default: Falling nodes and dropped items.
+This document describes how they behave and what you can do with them.
+
+## Falling node (`__builtin:falling_node`)
+
+Is created when a node with the special group `falling_node=1` starts to fall.
```

Comment:
Strictly spoken nodes can't fall, hence it might be misleading to mention that `a node [..] starts to fall`.

How about:
```
This entity is created by `minetest.check_for_falling` in place of a node
with the special group `falling_node=1`.
```